packages/client/src/transformer/index.ts

```diff
@@ -354,5 +354,8 @@ export const gqlToDraftTransformer = (
   if (queryData._fhirIDMap) {
     transformedData._fhirIDMap = queryData._fhirIDMap
   }
+  if (queryData.history) {
+    transformedData.history = queryData.history
+  }
```

Comment:
Instead of saving the history in data, [operationHistories](https://github.com/opencrvs/opencrvs-core/blob/2e1cfe0565ad8308c1d1bbc16db01ac788765fef/packages/client/src/applications/index.ts#L181) would be a better place in my opinion.